crl-test/project.tf

```diff
@@ -0,0 +1,20 @@
+module "crl-project" {
```

Comment:
I am in general against creating projects with TF. It's a simple enough thing to do manually, and is done infrequently enough that IMO it's not worth capturing in TF. Also it requires whatever is running the TF to have a much higher level of permissions than doing pretty much anything else in GCP that we usually do with TF.

Also I am not a fan of the current state of the `google-project` module:
- It will throw a bunch of TF syntax warnings because it has not been ported to TF12 syntax
- It (IMO) awkwardly and unnecessarily bundles service account IAM and Vault secrets along with project creation

If you really feel strongly about managing the project with TF, I recommend you not use this module, and create the project resource directly.

Reply:
Talked offline, no longer using terraform-shared/google-project module, still creating google project.